### ISSUE TITLE
Classification accuracy on ModelNet40

### ISSUE THREAD

**Issue by sheshap** [13952522076/pointsMLP#1]
Dear Author,

Thank you for releasing the code.
Following the code and instructions given in https://anonymous.4open.science/r/pointMLP-pytorch/README.md
I got a Classification accuracy of 93.4%
Using the pre-trained models I could reproduce 94.4%.

Please suggest how to reproduce the exact accuracy of 94.5%

Thanks in advance.

**Comment by ma-xu** [comment]
@sheshap 

Thanks for your interest.

The performance on ModelNet40 of almost all methods are not stable, see (https://github.com/CVMI-Lab/PAConv/issues/9#issuecomment-873371422).
If you run the same codes for several times, you will get different results.

Here is the log we got the 94.5 acc with voting: https://drive.google.com/file/d/1z-QbWLMlGxfMVPImcLeaDOVOJGvm64ny/view?usp=sharing

Also, the randomness of ModelNet40 is our motivation to experiment on ScanObjectNN benchmark, and report the mean/std results of several runs.

**Note** that this repo is not the offical realsed repo, just recording our development history. We will release an official repo later.

Please let me know if you have any questions or concerns.

Best,
Xu

**Comment by sheshap** [comment]
@13952522076 

Thank you so much for your quick reply.
I understand the randomness of ModelNet40.

Looking forward to the official repo release.
Very interesting work. Congratulations on the state-of-the-art results (both ModelNet40 and ScanObjectNN).

Also, a quick note. MVTN uses 2048 points as input to their model while other models in table 3 use 1024 points.

**Comment by ma-xu** [comment]
@sheshap 

Thanks for your kind words and your kind remind. We will update it in the final version.

Best,
Xu
